sections/Chapter 2 Problem Formulation/Semester_Context.tex

```diff
@@ -0,0 +1,26 @@
+\section{Semester Context}
+This semester is thematically based on students learning how to collaborate as teams on a complex project.
+
+To accommodate this, students work in interdependent teams on separate parts of a complex project. 
+Each team has a specific role, and should, to some degree, provide support to teams that they are collaborating with.
+To this, students are taught certain software process models, and are expected to use them in their work.
+This is supposed to emulate the environment of an actual software project.
+
+We are part of the Knox multi-project. In the next section, we will describe Knox and our role in more detail.
+% However, it is important to note that Knox is developed using Agile software development practices.
+% This is also the process model that we will be using in this semester.
+% 
+% The choice to use Agile software development practices is not a technical decision, but rather a choice of a process model that is more suitable for the type of work that we are doing.
+% We find the benefits of an Agile approach superior to other models, considering the scope, details, and requirements for the project.
+% To specify, we will be dealing with many of unknowns, which means that the flexibility of an agile approach will be beneficial.
+
+
+
+% A case could be made for the waterfall model, in that we have a certain deadline for delivering a report, and the phases of writing the report (and the deadline) are known beforehand. 
+% However, the contents of the report is what is important, and that will only become better by 1. following and learning from the designated process, and 2. doing good work, which the agile approach — in our estimation — would help us do.
```

Comment:
"which the agile approach should help us do." instead of "which the agile approach — in our estimation — would help us do."